parser.go

```diff
@@ -135,7 +140,7 @@ func (parser *Parser) ParseGeneralAPIInfo(mainAPIFile string) error {
 				case "@description":
 					if parser.swagger.Info.Description == "{{.Description}}" {
 						parser.swagger.Info.Description = strings.TrimSpace(commentLine[len(attribute):])
-					} else {
+					} else if multilineBlock {
```

Comment:
I think Below code insted of `multiLineBlock` more simple.
What do you think?
```
					if parser.swagger.Info.Description == "{{.Description}}" {
						parser.swagger.Info.Description = strings.TrimSpace(commentLine[len(attribute):])
					} else if previousAttribute == attribute {
						parser.swagger.Info.Description += "\n" + strings.TrimSpace(commentLine[len(attribute):])
					}
```

Reply:
It's a bit simpler, but I think it creates a higher chance of reintroducing the bug later with another tag. I wanted `multilineBlock` specifically because it moves the `previousAttribute == attribute` to the previous scope before the `switch`, and then anyone changing more tags in the future will notice it.

Up to you, I can simplify per your comment if you still think that's better.

Reply:
I see. I think good your idea.